packages/next/src/client/components/react-dev-overlay/internal/components/copy-button/index.tsx

```diff
@@ -84,12 +89,14 @@ export function CopyButton({
       type="button"
       title={title}
       aria-label={label}
-      disabled={isDisabled}
+      aria-disabled={isDisabled}
```

Comment:
should we also keep the `disabled={isDisabled}`? I added that before cause it's visually disabled

Reply:
Forgot to push CSS changes to keep it visually disabled by going from `:disabled` to `[aria-disabled='true']` in the CSS.

The `disabled` was causing the lost focus because `:disabled` elements are not focusable. See https://css-tricks.com/making-disabled-buttons-more-inclusive/ for more info.